core/pkg/model/kubemodel/volume.go

```diff
@@ -0,0 +1,19 @@
+package kubemodel
+
+import "time"
+
+// @bingen:generate:Volume
+type Volume struct {
```

Comment:
should we add an 'ephemeral volume' type as well? Jay what kinds of things do we want to track for ephemeral storage?